forc/src/ops/forc_build.rs

```diff
@@ -170,7 +171,16 @@ pub fn build(command: BuildCommand) -> Result<pkg::Compiled> {
         let hash_file_name = format!("{}{}", &manifest.project.name, SWAY_BIN_HASH_SUFFIX);
         let hash_path = output_dir.join(hash_file_name);
         fs::write(hash_path, &bytecode_hash)?;
-        info!("  Bytecode hash is: {}", bytecode_hash);
+        info!("  Hash of the Script bytecode is: {}", bytecode_hash);
+    }
+
+    if let TreeType::Predicate = compiled.tree_type {
```

Comment:
These two consecutive `if let` can probably be replaced with ` match` statement where we do nothing for `_`.